src/reconcile/rec_visibility.c

```diff
@@ -212,9 +212,15 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, v
          * point can move forward during reconciliation so we use a cached copy to avoid races when
          * a concurrent transaction commits or rolls back while we are examining its updates. As
          * prepared transaction IDs are globally visible, need to check the update state as well.
+         *
+         * We avoid the checkpoint transaction pinning the oldest txn id. Therefore, the
+         * r->las_running can move beyond checkpoint transaction id. Need to do a proper visiblity
```

Comment:
Could be worded better as: The checkpoint transaction doesn't pin the oldest txn id, therefore the r->last_running can move beyond the checkpoint transaction id.

Also last_running not las_running.

Reply:
Thanks, will reword that.